src/Differ/MapDiffer.php

```diff
@@ -157,6 +157,8 @@ private function getDiffForArrays( array $old, array $new ) {
 	/**
 	 * Returns if an array is associative or not.
 	 *
+	 * @SuppressWarnings(PHPMD.UnusedLocalVariable)
+	 *
```

Comment:
There is a fix for this issue in phpmd/phpmd#198 which will be in the next PMD release 2.2.0. I'm ok with merging this. I'd like to have a comment refering to the fix and explaining why it's currently present.